### PR TITLE
refactoring TelegramBotScanner.md

### PR DESCRIPTION
**Pull request by Joel-max-s**
i refactored the TelegramBotScanner.md - doc-file for better readability

## Description

I just added a code block, some code-statements and an unordered list for better readability of the file.
The contents where not changed

## Type of change

Please make your selection.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change requires a new dependency
- [x] Code Maintainability / comments
- [ ] Code Refactoring / future-proofing

## How Has This Been Tested?

I'ts just a documentation change, watched the new md-file in the preview

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added pytest unit tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

non of them does match, it`s just a little doc-update

### CODE REVIEW

File: docs/TelegramBotScanner.md [hunk 1]
```diff
@@ -8,25 +8,27 @@ For a more detailed explanation of how the scanner functions please read the fol
 
 https://playful-curio-e62.notion.site/Scanning-the-market-fd9b58b059dd4cf8addb167af7f36311
 
-If you have not already setup the Telegram Bot you will need it setup to use this function, please checkout the TelegramBot.md for setup instructions
+If you have not already setup the Telegram Bot you will need it setup to use this function, please checkout the [TelegramBotSetup.md](TelegramBotSetup.md) for setup instructions
 
 There is a new section added to the config file (this can be found in the config.json.sample).  The section looks like this:
 
+```
```
Comment (warrmr) [inline docs/TelegramBotScanner.md:15]: If you use ```json it will turn on syntax highlighting :).

**Review by markhollingworth-worthit — APPROVED** [review]

**Comment by whittlem** [comment]
Should this be merged or still in progress?

**Comment by Joel-max-s** [comment]
> Should this be merged or still in progress?

File: docs/TelegramBotScanner.md (hunk 1 above)
Comment (Joel-max-s) [reply docs/TelegramBotScanner.md:15]: I didn't use the ````json thing, because the example is not json-comform with the comment behind the value.

**Comment by warrmr** [comment]
> > Should this be merged or still in progress?
> 
> Oops i accidentaly closed the PR. (and opend it right after it again)
> 
> From my side it is ready. I didn't use the ````json thing, because the example is not json-comform with the comment behind the value.

Im down with that :)  I think you can use # in JSON for a comment but not all JSON parsers will accept comments in JSON so happy with the reasonings. :) 


@whittlem Happy to merge! :)

**Review by warrmr — APPROVED** [review]

**Pull request merged**